protocol.md

```diff
@@ -1078,13 +1078,24 @@ interface Hover {
 	contents: MarkedString | MarkedString[];
 	
 	/**
-	 * An optional range
+	 * An optional range is a range inside a text document 
+	 * that is used to visualize a hover, e.g. by changing the background color.
 	 */
 	range?: Range;
 }
 ```
 Where `MarkedString` is defined as follows:
 ```typescript
+/**
+ * The marked string is rendered:
+ * - as markdown if it is represented as a string
```

Comment:
Specifying default is markdown is IMO not making the protocol better.
If your patch is amending the current version of the protocol, then it should say that the marked string is *usually* rendered as markdown since it may not be the choice taken by all clients.
If your patch is a proposal for a future version of the protocol, then simply finding a way to avoid ambiguity (remove string and keep only MarkedString) seems much better.